ui_components/setup_wizard.py

```
@@ -28,8 +28,37 @@ def render_wizard(self):
         if 'wizard_completed' not in st.session_state:
             st.session_state.wizard_completed = False
         
-        # Show progress
-        self._show_progress()
+        # Add reconnect button if already connected
+        if st.session_state.get('connected', False):
+            col1, col2, col3 = st.columns([1, 2, 1])
```

Ruff (0.14.2)

Unpacked variable `col1` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)